projects/kit/components/input-count/input-count.style.less

```diff
@@ -32,10 +32,12 @@
     display: flex;
     flex-direction: column;
     margin-left: 2px;
+    height: 100%;
 }
 
 .button {
     && {
+        display: flex;
         width: calc(var(--tui-height-m) * 0.75);
         height: calc(50% - 1px);
 
```

Comment:
Got rid of this crap in Safari:
<img width="479" alt="image" src="https://user-images.githubusercontent.com/17404191/111897108-c9e91300-8a3f-11eb-9dce-c50351bb8348.png">

Reply:
Now works smoothly fine, btw:
<img width="474" alt="image" src="https://user-images.githubusercontent.com/17404191/111898132-edaf5780-8a45-11eb-92ab-503fa589efd7.png">